webdriver/tests/bidi/input/perform_actions/pointer_mouse.py

```diff
@@ -150,7 +150,6 @@ async def test_middle_click(bidi_session, top_context, load_static_test_page):
         actions=actions, context=top_context["context"]
     )
 
-
```

Comment:
I'm not a fan of this kind of code style changes, as we don't have styling rules in the project.

Reply:
Ack